docs/docsite/rst/dev_guide/style_guide/index.rst

```diff
@@ -238,7 +238,7 @@ Modules require different suffixes from other plugins:
 Adding local TOCs
 ^^^^^^^^^^^^^^^^^
 
-The page you're reading includes a `local TOC <http://docutils.sourceforge.net/docs/ref/rst/directives.html#table-of-contents>`_.
+The page you're reading includes a `local TOC <https://docutils.sourceforge.net/docs/ref/rst/directives.html#table-of-contents>`_.
```

Comment:
```suggestion
The page you're reading includes a `local TOC <https://docutils.sourceforge.io/docs/ref/rst/directives.html#table-of-contents>`_.
```

Reply:
I see the 304 redirect. As requested, updated those urls and committed the changes. Thank you!